package.json

```diff
@@ -21,7 +21,9 @@
     "demo:start": "builder concurrent demo:start:app demo:start:graphql-server",
     "type-check": "tsc",
     "lint": "tslint 'src/**/*.{ts, tsx}'",
-    "test": "jest"
+    "test": "jest",
+    "prettier": "prettier --write {example,src}/**/*.{js,ts,tsx}",
```

Comment:
NIT: There's also `https://github.com/FormidableLabs/urql/blob/master/coverage-fix.js` we probably should include. `lint-staged` glob already covers it.

Reply:
Good call!